forms.py

```diff
@@ -0,0 +1,33 @@
+from flask_wtf import FlaskForm
+from wtforms import StringField, PasswordField, SubmitField
+from wtforms.fields.html5 import EmailField
+from wtforms.validators import DataRequired, EqualTo, Length, ValidationError
+from models import User, Recipe
+from flask_login import current_user
+
+
+class registrationForm(FlaskForm):
```

Comment:
Tested. User needs to meet character requirements for the username, email, and password. Duplicate usernames and/or emails are not allowed.